packages/suite/src/components/suite/Settings/components/DeviceBanner.tsx

```diff
@@ -35,11 +46,21 @@ interface DeviceBannerProps {
 
 export const DeviceBanner = ({ title, description }: DeviceBannerProps) => {
     const { device } = useDevice();
+
+    const { transport } = useSelector(state => ({
+        transport: state.suite.transport,
+    }));
+
+    const showWebUsb = isWebUSB(transport);
+
     return (
         <Wrapper data-test="@settings/device/disconnected-device-banner">
             <DeviceBadge type="CONNECT" shape="CIRCLE" size={64} device={device} />
             <Column>
-                <P weight="bold">{title}</P>
+                <StyledP weight="bold">
+                    {title} {!device && showWebUsb && <WebusbButton icon="SEARCH" />}
```

Comment:
What about `device?.connected` condition to cover the situation when you have remembered device, but disconnected? And then it would be shown on /settings/cons page too.
```suggestion
                    {title} {!device?.connected && showWebUsb && <WebusbButton icon="SEARCH" />}
```

Reply:
I did it like that to avoid displaying it in `Crypto` section.

However, I will adjust the banner there to be ok.

Reply:
![Screenshot 2022-06-14 at 10 05 54](https://user-images.githubusercontent.com/33235762/173526543-c7aed380-85eb-4006-aea5-76027b90296d.png)

I believe that the first line is unnecessary there. I will remove it and move description into title

Reply:
![Screenshot 2022-06-14 at 10 14 04](https://user-images.githubusercontent.com/33235762/173528181-12932807-73ad-4259-8d88-e84ae8822814.png)
![Screenshot 2022-06-14 at 10 14 15](https://user-images.githubusercontent.com/33235762/173528216-240a88f0-6410-4cfa-bbc2-7f67764d7c5b.png)